tests/admin_scripts/tests.py

```diff
@@ -2123,6 +2125,30 @@ def test_custom_project_template_from_tarball_by_url(self):
         self.assertTrue(os.path.isdir(testproject_dir))
         self.assertTrue(os.path.exists(os.path.join(testproject_dir, 'run.py')))
 
+    def test_custom_project_template_from_tarball_by_url_uses_Django_ua(self):
+        """
+        User-Agent is set to 'Django/version' when obtaining tarball.
+        """
+        user_agent = 'undefined'
+
+        def serve_template(request, *args, **kwargs):
+            nonlocal user_agent
+            user_agent = request.headers['User-Agent']
+            return serve(request, *args, **kwargs)
+
+        import admin_scripts.urls as test_urls
+        test_urls.urlpatterns += [
+            path('ua_check/<path:path>', serve_template, {
+                 'document_root': os.path.join(test_urls.here, 'custom_templates')}),
+        ]
```

Comment:
Use hanging indentation as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Reply:
Not entirely sure I'm getting this as intended. Is this ok?
```
        urls.urlpatterns += [path(
                'ua_check/<path:path>',
                serve_template,
                {'document_root': os.path.join(urls.here, 'custom_templates')}
        )]
```